### PR TITLE
feature/remove-httpd-proxy

### PR DESCRIPTION
**Pull request by jrouzierinverse**
# Description
Remove httpd.proxy

# Impacts
If it was used migrate to use haproxy.

# Delete branch after merge
YES

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries
## Enhancements
* Remove httpd.proxy service

### CODE REVIEW

File: debian/patches/debianize.patch [hunk 1]
```diff
@@ -24,15 +24,6 @@ index 5492ed51f1..d2237bc88e 100644
  #
  # services.radiusd_acct
  #
-@@ -518,7 +518,7 @@ httpd_proxy=enabled
- # services.httpd_binary
- #
- # Location of the apache binary. Only necessary to change if you are not running the RPMed version.
--httpd_binary=/usr/sbin/httpd
```
Comment (fdurand) [inline debian/patches/debianize.patch]: Not related

**Review by fdurand — CHANGES_REQUESTED** [review]

**Comment by fdurand** [comment]
You have to remove the systemd script too

**Comment by jrouzierinverse** [comment]
> You have to remove the systemd script too

Would this be in the rpm/debian packaging?

File: debian/patches/debianize.patch [hunk 2]
```diff
@@ -83,23 +83,6 @@ index 59bbf12796..e06af92d10 100644
...
  # We want systemd to give httpd some time to finish gracefully, but still want
-diff --git a/conf/systemd/packetfence-httpd.proxy.service b/conf/systemd/packetfence-httpd.proxy.service
-index 8399433b77..078e1c5e64 100644
---- a/conf/systemd/packetfence-httpd.proxy.service
-+++ b/conf/systemd/packetfence-httpd.proxy.service
-@@ -10,9 +10,9 @@ After=packetfence-base.target packetfence-config.service packetfence-iptables.se
- StartLimitBurst=3
- StartLimitInterval=10
- Type=notify
--ExecStartPre=/usr/bin/rm -f /usr/local/pf/var/run/httpd.proxy.pid
```
Comment (fdurand) [inline debian/patches/debianize.patch:94]: The service is removed so there is no need to patch it

**Comment by fdurand** [comment]
interceptproxy.pm needs to be removed too

**Comment by jrouzierinverse** [comment]
done

**Pull request merged**